resolwe_bio/tools/icount_qc.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python3
+"""Parse STAR mapping results and report sample QC."""
+import argparse
+import json
+
+
+def parse_arguments():
+    """Parse command line arguments."""
+    parser = argparse.ArgumentParser(description="Run iCount demultiplexing.")
+    parser.add_argument('mapping_stats', help="Star mapping stats.")
+    return parser.parse_args()
+
+
+def report_qc(cdna, reads):
```

Comment:
I would avoid naming this quantity cDNA. cDNA scores for each position are computed in a complex process (`iCount xlsites`). cDNA is not equal to the sum of `umr + mlr`.